src/js/views/globals/root_views.js

```diff
@@ -113,6 +114,12 @@ const ModalRegionView = TopRegionView.extend({
   },
 });
 
+const ModalLoadingRegionView = ModalRegionView.extend({
+  contains() {
+    return true;
+  },
```

Comment:
This stops user clicks from closing the modal. Or doing anything else in the app while the modal is open.